test/screenshot/text-field/textArea.tsx

```diff
@@ -19,25 +21,28 @@ const textareaFields = textareaVariants.map((variant) => {
       return rtlMap.map((isRtl: {isRtl?: boolean}) => {
         return disabledMap.map((disabled: {disabled?: boolean}) => {
           return helperTextMap.map((helperText: HelperTextMapType | {}) => {
-            let helperTextKey = getHelperKeyText(
-              helperText,
-              isValidationMsg(helperText)
-            );
-            if (helperTextKey.length > 0) {
-              helperTextKey = `-${helperTextKey}`;
-            }
-            const props = Object.assign(
-              {},
-              variant,
-              dense,
-              disabled,
-              helperText,
-              isRtl
-            );
-            const key = `textarea-${JSON.stringify(props)}${helperTextKey}`;
-            return (
-              <TestField variant='textarea' {...props} key={key} id={key} />
-            );
+            return characterCounterMap.map((characterCounter: CharacterCounterMapType | {}) => {
```

Comment:
revert this too, and just add a case to the `test/screenshot/text-field/characterCounter.tsx` file.